Implement 'osnadmin channel info' subcommand and update tests #4890 (#4919)
hyperledger/fabric: docs/wrappers/osnadmin_channel_postscript.md

  Status 200 and the list of channels are returned.

- Using the `--channelID` flag to list more details for `mychannel`.

  ```
  osnadmin channel list -o orderer.example.com:9443 --ca-file $CA_FILE --client-cert $CLIENT_CERT --client-key $CLIENT_KEY --channelID mychannel

  Status: 200
  {

>   "name": "mychannel",
>   "url": "/participation/v1/channels/mychannel",
>   "consensusRelation": "consenter",
>   "status": "active",
>   "height": 3

Reviewer: ```suggestion
	"name": "mychannel",
	"url": "/participation/v1/channels/mychannel",
	"consensusRelation": "consenter",
	"status": "active",
	"height": 3
```

  }

  ```

  Status 200 and the details of the channels are returned.

### osnadmin channel info example

Here are some examples of the `osnadmin channel info` command.

> - Using the `--channel-id` flag to get detailed information for mychannel from the orderer at `orderer.example.com:9443`.

Reviewer: ```suggestion
- Using the `--channelID` flag to get detailed information for mychannel from the orderer at `orderer.example.com:9443`.
```

  ```

>   osnadmin channel info -o orderer.example.com:9443 --ca-file $CA_FILE --client-cert $CLIENT_CERT --client-key $CLIENT_KEY --channel-id mychannel

Reviewer: ```suggestion
  osnadmin channel info -o orderer.example.com:9443 --ca-file $CA_FILE --client-cert $CLIENT_CERT --client-key $CLIENT_KEY --channelID mychannel
```

  Status: 200
  {

>   "name": "mychannel",
>   "url": "/participation/v1/channels/mychannel",
>   "consensusRelation": "consenter",
>   "status": "active",
>   "height": 3

Reviewer: ```suggestion
	"name": "mychannel",
	"url": "/participation/v1/channels/mychannel",
	"consensusRelation": "consenter",
	"status": "active",
	"height": 3
```
